src/attack.c

```diff
@@ -506,6 +506,8 @@ attack_host (struct scan_globals *globals, struct in6_addr *ip, GSList *vhosts,
         }
       else if (plugin == NULL)
         break;
+      else if (plugin != NULL && plugin == PLUG_RUNNING)
+        usleep (50000);
```

Comment:
Could you add a comment about the time length? Maybe in miliseconds or seconds. it is not obvious how long this is